scripts/psl2tsv.sh

```diff
@@ -0,0 +1,88 @@
+if [ -z "$1" ]
+  then
+    echo "Specify location of psl files"
+    return
+fi
+
+if [ -z "$2" ]
+  then
+    echo "Specify assembly id"
+    return
+fi
+
+INPUT_FOLDER=$1
+ASSEMBLY_ID=$2
+echo 'Processing files in ' $1
+
+MATCHES_PSL=${INPUT_FOLDER}/matches.psl
+MATCHES_BED=${INPUT_FOLDER}/matches-split.bed
+EXACT_MATCHES=${INPUT_FOLDER}/mapping-temp.tsv
+INEXACT_MATCHES=${INPUT_FOLDER}/mapping-inexact-temp.tsv
+
+for i in {1..2}; do
```

Comment:
Why suddenly start mixing tabs and spaces here?

Reply:
Thanks, fixed